### PR TITLE
Remove giving player skull directly into Inventory

### PR DESCRIPTION
**Pull request by NeverGrowUp**
Removes the giving of a Skull directly into the killers inventory (which would often overwrite another inventory item), and relies on the skull being dropped onto the ground for killer to pick up.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,9 +122,9 @@
   <dependencies>
         <!-- Spigot -->
         <dependency>
-          <groupId>com.destroystokyo.paper</groupId>
+          <groupId>io.papermc.paper</groupId>
           <artifactId>paper-api</artifactId>
-          <version>${spigot.version}</version>
+          <version>1.17.1-R0.1-SNAPSHOT</version>
```
Comment (addstar) [inline pom.xml]: I think this was this changed accidentally?

File: src/main/java/au/com/addstar/truehardcore/TrueHardcore.java [hunk 2]
```diff
@@ -109,7 +109,7 @@
           Material.PODZOL,
           Material.GRASS,
           Material.GRASS_BLOCK,
-          Material.GRASS_PATH,
+          Material.DIRT_PATH,
```
Comment (addstar) [inline src/main/java/au/com/addstar/truehardcore/TrueHardcore.java:112]: :clap: Nice! I had forgotten about this change.. might be a problem in several plugins.

File: pom.xml (hunk 1 above)
Comment (NeverGrowUp) [reply pom.xml]: Correct. Should remain unchanged. Had initial problems due to PICNIC error.

**Comment by addstar** [comment]
I'd probably also remove the entire "findPlacer" lookup here, but it might take a bit more effort to clean up:

https://github.com/AddstarMC/TrueHardcore/blob/69d0620596ddf2c9cb1acae84f5f176ae53720c7/src/main/java/au/com/addstar/truehardcore/listeners/PlayerListener.java#L451

File: pom.xml [hunk 3]
```diff
@@ -121,9 +122,9 @@
   <dependencies>
         <!-- Spigot -->
         <dependency>
-          <groupId>com.destroystokyo.paper</groupId>
+          <groupId>io.papermc.paper</groupId>
           <artifactId>paper-api</artifactId>
-          <version>${spigot.version}</version>
+            <version>${spigot.version}</version>
```
Comment (addstar) [inline pom.xml]: Well.. you kinda fixed it :stuck_out_tongue_closed_eyes: can you also fix the extra whitespace you added? :wink:

**Review by addstar — APPROVED** [review]
:+1: Looks good! Nice work, and good testing!

**Comment by NeverGrowUp** [comment]
This commit fixes #14

**Pull request merged**